bait-files/src/test/java/com/google/sps/servlets/ReviewServletTest.java

```diff
@@ -155,4 +156,81 @@ public void doGet_whenReviewExistsAndRequestExists_returnsReviewAndRequest()
     String rawJsonResponse = stringWriter.toString();
     assertTrue(rawJsonResponse.startsWith("{\"request\":{\"requestId\":14,"));
   }
+
+  @Test
+  public void doPost_whenOneReviewCreated_StoresOneReview() throws IOException, ServletException {
+    ObjectifyService.ofy()
+        .save()
+        .entity(new Request(14L, "4", "human", "human47", "human47@gmail.com", "2930 pearl street",
+            "no_image", "555-555-5555", "some notes"))
+        .now();
+
+    HttpServletRequest request = mock(HttpServletRequest.class);
+    HttpServletResponse response = mock(HttpServletResponse.class);
+    Map<String, String[]> parameters = new HashMap<String, String[]>();
+    parameters.put("review-request-id", new String[] {"14"});
+    parameters.put("review-user-id", new String[] {"4"});
+    parameters.put("name-validity", new String[] {"valid"});
+    parameters.put("username-validity", new String[] {"valid"});
+    parameters.put("email-validity", new String[] {"invalid"});
+    parameters.put("phone-validity", new String[] {"invalid"});
+    parameters.put("address-validity", new String[] {"valid"});
+    parameters.put("image-validity", new String[] {"invalid"});
+    parameters.put("authenticity-rating", new String[] {"2"});
+    parameters.put("reviewer-notes", new String[] {"Look Like You Know - Royal Blood"});
+
+    when(request.getParameterMap()).thenReturn(parameters);
+    when(request.getParameter("status")).thenReturn("COMPLETED");
+
+    StringWriter stringWriter = new StringWriter();
+    PrintWriter writer = new PrintWriter(stringWriter);
+    when(response.getWriter()).thenReturn(writer);
+
+    new ReviewServlet().doPost(request, response);
+
+    Query<Review> query = ObjectifyService.ofy().load().type(Review.class);
+    List<Review> allReviews = query.list();
+
+    String rawJsonResponse = stringWriter.toString();
```

Comment:
I think you can delete this line since you don't seem to be doing anything with it =)